xml/System.IO/StreamReader.xml

```diff
@@ -1513,10 +1513,12 @@ Following a call to <xref:System.IO.StreamReader.Close%2A>, any operations on th
         <Parameter Name="buffer" Type="System.Span&lt;System.Char&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">When this method returns, contains the specified span of characters replaced by the characters read from the current source.</param>
+        <summary>Reads the characters from the current stream into a span.</summary>
+        <returns>The number of characters that have been read, or 0 if at the end of the stream and no data was read. The number will be less than or equal to the <paramref name="buffer" /> length, depending on whether the data is available within the stream.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.IO.IOException">The number of characters read from the stream is larger than the <paramref name="buffer" /> length.</exception>
+        <exception cref="T:System.ArgumentNullException"><paramref name="buffer" /> is <see langword="null" />.</exception>
```

Comment:
> _to avoid having the CI touching this without a "real" update_

Would you mind elaborating? What do you mean by that, @mairaw?

Reply:
When I run an update, let's say when I run the CI to include the 3.1 assemblies, it would move the paramref to a new line automatically.

See an example here: https://github.com/dotnet/dotnet-api-docs/commit/64dcbe46ec8f7db0c6d5957ebff03d94c8fa6c66#diff-ab0ccb189b0e628c7124880d23b154af

So to avoid having files touched by the CI process which don't really have updates for a certain version, I've been keeping the files in the format it expects. Makes sense?

But I think we could log a customer suggestion so mdoc doesn't do that anymore and see what @joelmartinez says. Makes sense?

Reply:
Got it! I did the same with the other exception that started with a paramref.